build/templates/_converters.py.mako

```diff
@@ -8,6 +8,32 @@ import datetime
 import six
 
 
+def _string_to_list(string, prefix):
+    '''Convert a string to a list of repeated capabilities numbers I.e. no prefix
+
+    '0,1,3' becomes [0, 1, 3]
+    '0-2' becomes [0, 1, 2]
+    'ScriptTrigger0,ScriptTrigger1' becomes [0,1]
+    '''
+    channels = string.replace(prefix, '').split(',')
+    channels_list = []
+    for c in channels:
+        c = c.strip()
+        if len(c.split('-')) > 1:
+            assert len(c.split('-')) == 2, "Only one '-' allowed. {0}".format(c)
+            start, end = c.split('-')
+            if int(end) < int(start):
+                for i in range(int(start), int(end) - 1, -1):
+                    channels_list.append(str(i))
+            else:
+                for i in range(int(start), int(end) + 1):
+                    channels_list.append(str(i))
+        else:
+            channels_list.append(c)
+
+    return channels_list
+
+
 def convert_repeated_capabilities(repeated_capability, prefix=''):
```

Comment:
add docstring on what this function does and returns.

Reply:
Added